go-controller/pkg/util/net_linux.go

```diff
@@ -92,8 +92,9 @@ func (defaultNetLinkOps) AddrDel(link netlink.Link, addr *netlink.Addr) error {
 	return netlink.AddrDel(link, addr)
 }
 
+// AddrAdd don't fail if the IP already exists
 func (defaultNetLinkOps) AddrAdd(link netlink.Link, addr *netlink.Addr) error {
-	return netlink.AddrAdd(link, addr)
+	return netlink.AddrReplace(link, addr)
```

Comment:
The wrapper function name should match the function that it's calling. Also, there may be some callers of `util.AddrAdd()` that explicitly don't want the replace semantics; I remember this coming up before.

Reply:
yeah, I know , it was an ugly hack to keep moving, I have to add the new method

Reply:
I removed this, it was the for loop using the variable as reference that was adding the same ip twice :face_with_head_bandage: